docs/user_guide.md

```diff
@@ -688,10 +688,10 @@ new ID token and the CLI secret will work again.
 Robot Accounts are accounts created by project admins that are intended for automated operations. They have the following limitations:
 
 1, Robot Accounts cannot login Harbor portal
-2, Robot Accounts can only perform `docker push`/`docker pull` operations with a token.
+2, Robot Accounts can only perform `docker push` / `docker pull` operations with a token.

```

Comment:
Could we rephrase to robot accounts can only perform operations using `docker` and `helm` CLI?  What we want to imply is that it's not eligible for API calls currently

Reply:
agreed